docker/bigdl/bigdl.yaml

```diff
@@ -0,0 +1,58 @@
+apiVersion: apps/v1
```

Comment:
rename as deployment.yaml
add howto run to readme

Reply:
> rename as deployment.yaml add howto run to readme

ok